pkg/platform/model/vcs.go

```diff
@@ -82,10 +82,8 @@ const (
 	// NamespaceCamelFlagsMatch is the namespace used for passing camel flags
 	NamespaceCamelFlagsMatch = `^camel-flags$`
 
-	// NamespaceSharedMatch is the namespace used for shared requirements (usually runtime libraries)
-	NamespaceSharedMatch = `^shared$`
-
-	NamespaceOrgSharedMatch = `^\w+\/shared(?$|\/\w+$)`
+	// NamespaceOrgMatch is the namespace used for org specific requirements
+	NamespaceOrgMatch = `^org\/(\w+)$`
```

Comment:
Can org names have dashes or any other punctuation?

Reply:
I don't know, but we probably shouldn't care either way. I'll drop the suffix matching.